styles/field.tss

```diff
@@ -1,26 +1,29 @@
 ".row[platform=ios]": {
-  selectionStyle: Ti.UI.iPhone.TableViewCellSelectionStyle.NONE,
-
-  layout: 'horizontal'
```

Comment:
TextField width on Android filling the row, don't know why, not layout is composite, #control left now is 160